src/main/java/core/basesyntax/service/impl/CsvFileWriterServiceImpl.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.CsvFileWriterService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class CsvFileWriterServiceImpl implements CsvFileWriterService {
+    @Override
+    public void writeToFile(String data, String filePath) throws IOException {
```

Comment:
It's a good practice to use try-with-resources for IO operations to ensure that resources are closed properly. Consider wrapping the `Files.write` operation within a try-with-resources block or ensure that resources are closed in case of an exception.

Reply:
Implement this comment.